packages/eslint-plugin-x/package.json

```
@@ -44,6 +43,10 @@
     "eslint-plugin-vue": "~8.7.1",
     "eslint-plugin-vuejs-accessibility": "~2.0.0"
   },
+  "devDependencies": {
+    "eslint": "~8.30.0",
```

This is going to conflict with dependabot PRs
```suggestion
    "eslint": "~8.31.0",
```